Kibetchirchir/send-itv2
# #161993537 build on Chore database config and travis.yml test

**What does this PR do?**
This PR configures our database 

Description of Task to be completed?
Create a database
Create a config db file
Intergrate to travis

How should this be manually tested?
Clone the repository,cd
run the command source .env(you should create the .env)
open your virtual enviroment and install requirements __pip install -r requirements.txt__
Create the database send-it
run pytest --cov app

What are the relevant pivotal tracker stories?
#161993537
